## OPCODE 129
uint8 baudcode
##  0 is    300 BPS
##  1 is    600 BPS
##  2 is   1200 BPS
##  3 is   2400 BPS
##  4 is   4800 BPS
##  5 is   9600 BPS
##  6 is  14400 BPS
##  7 is  19200 BPS
##  8 is  28800 BPS
##  9 is  38400 BPS
## 10 is  57600 BPS
## 11 is 115200 BPS
